libraries/dynamic-ui-react/src/style/abstracts/variables/_forms.scss

```
@@ -45,7 +45,7 @@ $input-disabled-border-color: var(--#{$prefix}gray-300) !default;
 $input-color: var(--#{$prefix}gray) !default;
 $input-border-color: var(--#{$prefix}gray-300) !default;      // unused
 $input-border-width: $input-btn-border-width !default;          // unused
-$input-box-shadow: $box-shadow-inset !default;                  // unused
+
```

por acá lo mismo que comentaba arriba de la variable, dejemos la definición que propone bootstrap que es la que igual va a tomar el ouput si no hay variable :) 

```suggestion
$input-box-shadow: var(--#{$prefix}box-shadow-inset) !default; // unused
```